jsx/FilterForm.js

```diff
@@ -127,7 +127,7 @@ class FilterForm extends React.Component {
     }
     if (key) {
       filter[key] = {};
-      if (value.length === 0) {
+      if (Array.isArray(value) && value.length === 0) {
```

Comment:
need to check that value is type string OR type object so change this to...
if ((typeof (value) === 'string' || typeof (value) === 'object') && value.length === 0)

Reply:
i think this fixes the filtering error we found above!